### ISSUE TITLE
importing large files in audapolis fails

### ISSUE THREAD

**Issue by anuejn** [audapolis/audapolis#355]
the error is similiar to https://stackoverflow.com/questions/62934017/file-size-6003804160-is-greater-than-possible-buffer-2147483647-in-node

**Comment by pajowu** [comment]
This might be a good moment in general to think about how we store media files. At the moment they are always in ram, which is not ideal i think

**Comment by larriant** [comment]
I'm also hitting this problem. How much work is it to fix do you think? Happy to dive and try and come up with a fix if you think it's possible and not too challenging.

**Comment by pajowu** [comment]
Yes, currently we do not have a fix for that and sadly development is slowed down at the moment. I think a simple solution (creating a proxy-media with a lower quality) could be implemented, but might lead to some langer changes (although they should be limited to a few files). If you are interested in trying to implement that, feel free to comment again. I would then write up a bit more on that.

**Comment by anuejn** [comment]
Personally I woulnt go with proxy media but directly with implementing file acces via the backend. To do that I would introduce a backend http endpoint that serves files from the filesystem and implements http seek operations. I guess this is less work and would also have a better user experience. There are some security footguns but I think we can build a safe solution.
